src/platform/qt/Log.cpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2013-2025 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "Log.h"
+
+#include <QLoggingCategory>
+
+mLOG_DEFINE_CATEGORY(QT, "Qt", "platform.qt");
+
+using namespace QGBA;
+
+Log* Log::s_target = nullptr;
+
+Log::Stream Log::log(int level, int category) {
+	return Stream(s_target, level, category);
+}
+
+void Log::setDefaultTarget(Log* target) {
+	s_target = target;
+}
+
+Log::Log() {
+	// Nothing to do
+}
+
+Log::~Log() {
+	if (s_target == this) {
+		s_target = nullptr;
+	}
+}
+
+void Log::postLog(int level, int category, const QString& string) {
+	QLoggingCategory cat(mLogCategoryName(category));
+	switch (level) {
+	case mLOG_DEBUG:
+	case mLOG_STUB:
+		qCDebug(cat).noquote() << string;
+		return;
+	case mLOG_INFO:
+		qCInfo(cat).noquote() << string;
+		return;
+	case mLOG_ERROR:
+	case mLOG_GAME_ERROR:
```

Comment:
`mLOG_GAME_ERROR` shouldn't be qCCritical. It won't ever be emitted by Qt logging anyway, but if anything it should be qCDebug.

Reply:
Done.